packages/riipen-ui/src/components/Chip.test.jsx

```diff
@@ -0,0 +1,136 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import Chip from "./Chip";
+
+it("renders without errors", () => {
+  try {
+    mount(<Chip />);
+  } catch (error) {
+    expect(true).toEqual(false);
+  }
+});
+
+describe("default props", () => {
+  it("sets correct default props", () => {
+    const defaultProps = new Chip().type.defaultProps;
+    defaultProps.classes.push("riipen-chip", "riipen");
```

Comment:
Does this actually do anything? Just looking at this test it's hard to tell exactly what this is doing. I am probably just stupid though.

Reply:
removed test